survival4d/nn/torch/__init__.py

```diff
@@ -56,9 +66,6 @@ def train_nn(xtr, ytr, batch_size, n_epochs, model_name, lr_exp, alpha, weight_d
         pbar = tqdm(enumerate(dataloader))
         for idx, (x, e, t) in pbar:
             # sort x (B, 11514), e (B, 1), t (B,) according to t
-            sort_idx = torch.argsort(t, descending=True, dim=0)
-            x = x[sort_idx, :]
-            e = e[sort_idx, :]
```

Comment:
Why not sort a cuda tensor? if the concern is speed, I wouldn't worry about that. The data is too small to have a significant impact.